src/app/shared/markdown-wrapper/markdown-wrapper.component.html

```diff
@@ -0,0 +1 @@
+<div fxFlex [innerHTML]="cleanedData"></div>
```

Comment:
Should be fine, double sanitize + innerHTML also santizes.

Reply:
1. Yes, the markdown service allows us to compile markdown directly to html, so we can pass it to DOMPurify. The reason I removed the ngx-markdown component was to avoid the second compilation step, since the input is already HTML at this point.